cmd/kpod/import.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/projectatomic/libpod/libpod"
+
+	"github.com/opencontainers/image-spec/specs-go/v1"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	importFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "change, c",
+			Usage: "Apply imgspecv1 configurations to the created image",
```

Comment:
Docker says:
  -c, --change list      Apply Dockerfile instruction to the created image (default [])

I have no idea what a imgspecv1 configurations are.


Man page says
```
       -c, --change=[]
          Apply specified Dockerfile instructions while importing the image
          Supported Dockerfile instructions: CMD|ENTRYPOINT|ENV|EXPOSE|ONBUILD|USER|VOLUME|WORKDIR
```

Reply:
Change should be a stringsliceflag.  You can specify it multiple times.

Reply:
Yeah, I wasn't sure if we wanted to call it docker file instructions. The struct that holds all these configurations is the imgspecv1.Image struct so I used that.
Should I rephrase it to be dockerfile instructions or?

Reply:
Well instead of saying Dockerfile, why not list the possible options.

Reply:
fixed